src/database/database.ts

```diff
@@ -90,6 +90,16 @@ export class BotDatabase {
         await this.database.run('CREATE INDEX IF NOT EXISTS "" ON JoinList (Allowed)')
                            .catch((err) => { this.log.error('Could not create index for JoinList table!'); });
 
+        await this.database.run(`CREATE TABLE IF NOT EXISTS TagList (
+                                    TagName         TEXT,
+                                    TagResponse     TEXT,
+                                    GuildID         TEXT);`)
+                                    .catch((err) => { this.log.error('Could not create TagList table!'); });
+
+        /* TODO: Index TagList?

```

Comment:
other than using indexes, maybe we could use compound keys based on TagName and GuildID, this would both index the table and disallow duplicates, also GuildID as first column would be better since it is consistent with other tables